### ISSUE TITLE
Upload multiple files

### ISSUE THREAD

**Issue by koenvanwijk** [microsoft/azurechat#282]
It seems the backend supports multiple files uploaded. However it does not seem to be possible in the UI.

**Comment by Masumi-Morishige** [comment]
Try uploading the 2nd file in the upload file icon :)
![CleanShot 2023-12-21 at 16 27 51@2x](https://github.com/microsoft/azurechat/assets/123607650/29c30347-4506-4957-84e6-23f157071a2b)

I guess it will RAG both file.

**Comment by koenvanwijk** [comment]
Yes indeed, I think the upload button also works multiple times. However selecting 10 files in de file selector would be nice.
